app/tests/Unit/CoreGameLogic/Feature/Spielzug/SpielzugCommandHandlerTest.php

```diff
@@ -854,6 +856,63 @@
     });
 });
 
+describe('handleDoMinijob', function () {
+
+    it('returns money to the player once after doing the minijob', function () {
+
+        $minijobs = [
+            "testMinijob" => new MinijobCardDefinition(
+                id: new CardId('textMinijob'),
+                pileId: PileId::MINIJOBS_PHASE_1,
+                title: 'Softwaretester',
+                description: 'Du arbeitest nebenbei als Softwaretester und bekommst einmalig Gehalt',
+                resourceChanges: new ResourceChanges(
+                    guthabenChange: new MoneyAmount(+500),
+                ),
+            )];
+
+        $this->shuffle($minijobs, PileId::MINIJOBS_PHASE_1);
+        $stream = $this->coreGameLogic->getGameEvents($this->gameId);
+        expect(PlayerState::getGehaltForPlayer($stream, $this->players[0]))->toBe(2000);
```

Comment:
Hier erwarten wir den Initialen Wert (50k) -> am besten gleich in die Configuration schreiben und dann hier `Configuration::Startkapital` verwenden.
```suggestion
        expect(PlayerState::getGehaltForPlayer($stream, $this->players[0])->value)->toBe(Configuration::Startkapital);
```